frontend/src/features/textView/useTextPage.ts

```diff
@@ -38,29 +26,33 @@ interface UseTextPageReturn {
   isFetching: boolean;
   isFetchingNextPage: boolean;
   isFetchingPreviousPage: boolean;
-  sourceLanguage: SourceLanguage;
+  dbLanguage: DbLanguage;
+  error: Error | null;
 }
 
 export function useTextPage(): UseTextPageReturn {
-  const { sourceLanguage, fileName, queryParams, defaultQueryParams } =
-    useDbQueryParams();
+  const { dbLanguage, fileName } = useDbRouterParams();
   const { isFallback } = useSourceFile();
 
   useSetDbViewFromPath();
 
+  const requestBodyBase = useStandardViewBaseQueryParams();
+
+  const searchParams = useSearchParams();
+  const active_segment =
```

Comment:
this does not need to have a default, I think (unless the BE API now expects it?). Setting it to "none" is probably why the middle view is not closing, so if "none" is the new empty state, we should update that closing logic as well.